packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -69,7 +69,7 @@ export function useHiddenSelect<T>(props: AriaHiddenSelectProps, state: SelectSt
     },
     inputProps: {
       type: 'text',
-      tabIndex: modality == null || state.isFocused || state.isOpen ? -1 : 0,
+      tabIndex: modality == null || state.isFocused || state.isOpen || isDisabled ? -1 : 0,
```

Comment:
Hmm. Why would the tabIndex matter if the input is also disabled (line 75)?

Reply:
This is because FocusScope has `focusableElements.push('[tabindex]:not([tabindex="-1"])')`. This makes the [matches selector](https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/focus/src/FocusScope.tsx#L177) resolve to true for disabled inputs w/ tabindex != -1

Reply:
I could add a `:not([disabled])` to the `focusableElements.push` line instead if that is more appropriate aka `focusableElements.push('[tabindex]:not([tabindex="-1"]):not([disabled])')` and remove this tabIndex change. Or I could change it so the inputProps tabIndex is undefined if disabled

Reply:
Yeah I think this is a FocusScope bug. Did Michael's PR fix this by any chance?

Reply:
If you are talking about https://github.com/adobe/react-spectrum/pull/1493, then no it didn't fix it. I'll go ahead and add the `:not([disabled])` bit to FocusScope